reactioncommerce/reaction
# fix: taxes no updating reactively in starterkit

Resolveshttps://github.com/reactioncommerce/reaction-next-starterkit/issues/469
Impact: **minor**  
Type: **bugfix**

## Issue
When changing the quantity of the items or shipping address, the tax calculated is not updated reactively, the page has to be refreshed to get the taxes.

## Solution
Return an updated Cart to the client after `appEvents` has run. We were previously returning the same `Cart` object as before it had run, therefore not getting any of the updates.

## Breaking changes
None

## Testing
1. Start up reaction platform - most of the testing is in the starterkit
1. Go to Admin, enable "Example tax service"
1. Create a new product.
1. Add it to the cart, go to the checkout page, see that taxes are calculated.
1. Change the quantity of the item. See that taxes are recalculated.
